test/test_models.py

```diff
@@ -682,10 +682,11 @@ def test_classification_model(model_fn, dev):
     model_name = model_fn.__name__
     if SKIP_BIG_MODEL and is_skippable(model_name, dev):
         pytest.skip("Skipped to reduce memory usage. Set env var SKIP_BIG_MODEL=0 to enable test for this model")
-    if model_name == "vit_h_14" and dev == "cuda":
+    if model_name == "resnet101" and dev == "cuda":
```

Comment:
Not sure if `vit_h_14` was fixed but regardless, we'll have to change or update the `xfail` URL below for resnet if we decide to skip this test

Reply:
HI @NicolasHug yes totally. Running test now. Will create new issue regarding resnet 101